packages/gatsby-plugin-image/README.md

```diff
@@ -26,331 +32,225 @@ If you're using the new `GatsbyImage` in addition to `StaticImage`, you'll also
 module.exports = {
   plugins: [
     `gatsby-plugin-image`,
+    `gatsby-psource-filesystem`,
     `gatsby-plugin-sharp`,
-    // `gatsby-transformer-sharp`
+    // `gatsby-transformer-sharp` // Needed for dynamic images
   ],
 }
 ```
 
-# StaticImage
+## Using the Gatsby Image components
 
-This component is a new, simpler way to use Gatsby's image processing tools and components without needing to write GraphQL queries. It is designed for static images such as logos rather than ones loaded dynamically from a CMS.
+### Decide which component to use
 
-```js
-import React from "react"
-import { StaticImage } from "gatsby-plugin-image"
+The Gatsby Image plugin includes two image components: one for static and one for dynamic images. The simplest way to decide which you need to is to ask yourself: _"will this image be the same every time the component or template is used?"_. If it will always be the same, then use `StaticImage`. If it will change, whether through data coming from a CMS or different values passed to a component each time you use it, then it is a dynamic image and you should use the `GatsbyImage` component.
 
-export const Dino = () => (
-  <StaticImage height={100} src="trex.png" alt="T-Rex" />
-)
-```
+### Static images
 
-The `src` prop is relative to the source file, like in static HTML.
+If you are using an image that will be the same each time the component is used, such as a logo or front page hero image, you can use the `StaticImage` component. The image can be either a local file in your project, or an image hosted on a remote server. Any remote images are downloaded and resized at build time.
 
-You can pass the same options as those available via [`gatsbyImageData`](#graphql-resolver) queries:
+1. **Add the image to your project.**
 
-```js
-import React from "react"
-import { StaticImage } from "gatsby-plugin-image"
-
-export const Dino = () => (
-  <StaticImage
-    src="trex.png"
-    placeholder="none"
-    layout="constrained"
-    width={200}
-    alt="T-Rex"
-    transformOptions={{ grayscale: true }}
-  />
-)
-```
+   If you are using a local image, copy it into the project. A folder such as `src/images` is a good choice.
```

Comment:
Should we mention anything about source-fileystem here?

Reply:
You don't need source-filesystem for this case. It is, counter-intuitively, only needed for remote images (because we need createRemoteFileNode)

Reply:
We import `gatsby-source-filesystem/create-file-node` for the createFileNode case.